lib/app/view/app_scaffold.dart

```diff
@@ -23,9 +26,11 @@ class AppScaffold extends StatelessWidget {
         elevation: 0,
       ),
       // TODO(Ryan): Render the bottom navigation bar for mobile
```

Comment:
Since we are rendering the bottom navigation bar for mobile, can you please remove this.